tasks/prepare-fbc-release/tests/test-prepare-fbc-release.yaml

```diff
@@ -0,0 +1,146 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-prepare-fbc-release 
+spec:
+  description: |
+    Run the prepare-fbc-release task and verify
+    that the task succesfully update the snapshot
+    with the ocpVersion, fromIndex, targetIndex,
+    and binaryImage for all the component in snapshot  
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      taskSpec:
+        steps:
+          - name: add-snapshot
+            image: quay.io/hacbs-release/release-utils:a918b3a7cbd40add5955a2f1acb4c3dbb29c44e9
+            script: |
+              #!/usr/bin/env sh
+              set -eux
+
+              cat > $(workspaces.data.path)/data.json << EOF
+              {
+                "fbc": {
+                  "fromIndex": "registry-proxy.engineering.redhat.com/rh-osbs/iib-preview-rhtap:v4.09",
+                  "targetIndex": "quay.io/redhat/redhat----preview-operator-index:v4.10",
+                  "binaryImage": "registry.redhat.io/openshift4/ose-operator-registry:v4.11"
+                }
+              }
+              EOF
+
+              cat > $(workspaces.data.path)/snapshot_spec.json << EOF
+              {
+                  "application": "foo-app",
+                  "artifacts": {},
+                  "components": [
+                      {
+                          "containerImage": "quay.io/hacbs-release-tests/test-ocp-version/test-fbc-component@sha256:f6e744662e342c1321deddb92469b55197002717a15f8c0b1bb2d9440aac2297",
```

Comment:
I don't think the test should rely on fetching stuff from Quay like this. The `skopeo` call should be mocked and a prepared json string should be returned.

Reply:
This is still valid.

Reply:
Added, thanks for the suggestion, PTAL :)